tests/app.rs

```diff
@@ -0,0 +1,49 @@
+use binsider::{app::Analyzer, error::Result, file::FileInfo, prelude::Event};
+use std::{fs, path::PathBuf, sync::mpsc};
+
+fn get_test_path() -> PathBuf {
+    PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("target")
+        .join("debug")
+        .join(env!("CARGO_PKG_NAME"))
```

Comment:
I think we can now use `CARGO_BIN_EXE_binsider` here:

> The absolute path to a binary target’s executable. This is only set when building an [integration test](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#integration-tests) or benchmark.

More info: https://doc.rust-lang.org/cargo/reference/environment-variables.html

Can you try it out? 😊

Reply:
Indeed, it works fine. Pushed.